cli.js

```diff
@@ -144,6 +143,19 @@ const promptEngines = () => {
 			status.engine = engine;
 			status.version = version;
 		}
+		else {
+			const wantsHelp = arg === '--help' || arg === '-h';
+			if (!wantsHelp) {
+				console.error('\nUnrecognized argument: ' + JSON.stringify(arg) + '\n');
+			}
+			console.log('[--os=(' + osChoices.map(choice => choice.value).join('|') + ')]')
+			console.log('[--engines=(' + engineChoices.map(choice => choice.value).join('|') + '),...]')
+			console.log('[<engine>@<version>]')
```

Comment:
I’d like to move this one to the top. Rationale: the `--os` and `--engines` are only useful/supposed to be used for CI environments, while `jsvu v8@7.4` is useful in non-scripted scenarios as well.